seshdash/utils/model_tools.py

```diff
@@ -153,3 +157,63 @@ def get_site_measurements(site):
             site_measurements_items.pop(i)
 
     return site_measurements_items
+
+def get_quick_status(sites):
+    """
+    returns battery state and future forecast
+    """
+    key = "8eefab4d187a39b993ca9c875fef6159"
+
+    #battery rules limits
+    battery_limit = Status_Rule.battery_rules.keys()
+    battery_limit.sort()
+
+    #weather rules limits
+    weather_limit = Status_Rule.weather_rules.keys()
+    weather_limit.sort()
+
+    results = []
+
+    #getting latest_points
+    for site in sites:
+        site_dict = {}
+
+        site_dict['site'] = site
+        #site coordinates
+        lat = site.position.latitude
+        lng = site.position.longitude
+
+        #forecast instance
+        instance = ForecastAPI(key,lat,lng)
```

Comment:
You can assume the weather data is already in the weather model. You don't need to fetch it here. It's automatically fetched in tasks.py